src/inim.nim

```diff
@@ -258,6 +257,18 @@ proc doRepl() =
     if currentExpression in ["exit", "exit()", "quit", "quit()"]:
         cleanExit()
 
+      if currentExpression in ["help", "help()"]:
```

Comment:
Here's the badly-indented code. 

`nimble build && git commit` :laughing:

Reply:
Whoops, buggered up the merge. Will fix